### ISSUE TITLE
Code Table Request - identifiers - published identifier

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#6431]
## Initial Request

_**Goal**: Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step._

[published identifier](https://arctos.database.museum/info/ctDocumentation.cfm?table=CTCOLL_OTHER_ID_TYPE#published_identifier) is nonsense and should be merged into "identifier."

Data:
[temp_published_identifier.csv.zip](https://github.com/ArctosDB/arctos/files/11835924/temp_published_identifier.csv.zip)

Summary:

```

 guid_prefix  | numrecs 
--------------+---------
 ALMNH:Paleo  |      17
 DMNS:Mamm    |       1
 DMNS:Para    |      27
 MSB:Mamm     |       2
 NMMNH:Paleo  |      12
 UAM:Ento     |      20
 UAM:ES       |       6
 UCM:Obs      |       2
 UMNH:Mamm    |       1
 UTEP:ES      |       3
 UTEP:Herp    |      11
 UTEPObs:Ento |       2
 UTEPObs:Herp |     106
```

Users:

 @ebraker
 @Nicole-Ridgwell-NMMNHS
 @campmlc
 @kderieg322079
 @DerekSikes
 @mvzhuang
 @AdrienneRaniszewski
 @aklompma
 @acdoll
 @jldunnum
 @jrdemboski

@ArctosDB/arctos-code-table-administrators 

---------------

## Approval

_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [x] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [x] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection

_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation

_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

_Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel._

_Make changes as described above. Ensure the URL of this Issue is included in the definition._

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
I usually do use erroneous citation for this type of thing, however in this case I'd have to create a specimen record to do it, because our numbers don't go up to 500000. I'd prefer not to create the specimen record because I think that would only lead to the error perpetuating.

**Comment by ebraker** [comment]
OK to merge for UCM

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Ours have been fixed manually.

**Comment by campmlc** [comment]
I fixed MSB's records. One of them was a BoLD BIN ID - which we don't have in the code table. I created BoLD as an agent, and added the issued by. But we should discuss whether we want to add the BoLD BIN identifier. This same record also has a BOLD Process ID - which is in the code table, but not obvious to find at BoLD. The processid is apparently the BoLD equivalent of a specimen record, which since it is a NEON example, doesn't link to the museum voucher at MSB. Bold is a mess. 
We should discuss in a separate issue what to do with BoLD's multiple identifier types. Similar situation at GenBank. But for now, I added "BIN number" in remarks and assigned the issued by BoLD, so this is OK for MSB.

**Comment by dustymc** [comment]
> discuss

There is already a robust solution in place: simply use the full identifier (``https://v3.boldsystems.org/index.php/Public_BarcodeCluster?clusteruri=BOLD:AAA3869``) as the value of type 'identifier' and everything will function as it should.

**Comment by campmlc** [comment]
See #6454.

**Comment by Jegelewicz** [comment]
UGH see also https://arctos.database.museum/guid/DMNS:Mamm:10191

which includes the determiner as [BOLD Engine ID](https://arctos.database.museum/agent/21259274), but the BOLD record says the identification is Chordata, not [Perognathus flavus](https://arctos.database.museum/name/Perognathus%20flavus)

http://boldsystems.org/index.php/Public_RecordView?processid=NEONU1696-13
